cfpb/consumerfinance.gov
# Migrate layout-full to CSS grid

## Additions

- Added `u-layout-grid_breakout` class for blocks that bleed out of the CSS grid to the browser window edge.

## Changes

- Add block around /consumer-tools/retirement/before-you-claim/about/ content area to fix problem where content bottom was erroneously flush.


## How to test this PR


1. These pages use the layout full template. They should not be broken:

Form explainer:
http://localhost:8000/owning-a-home/loan-estimate/

CCDB
http://localhost:8000/data-research/consumer-complaints/search/

Gov-delivery
https://www.consumerfinance.gov/govdelivery-subscribe/success/
https://www.consumerfinance.gov/govdelivery-subscribe/error/

Know Before You Owe
http://localhost:8000/know-before-you-owe/
http://localhost:8000/know-before-you-owe/compare/
http://localhost:8000/know-before-you-owe/timeline/

Rate Explorer
http://localhost:8000/owning-a-home/explore-rates/

Transcript
http://localhost:8000/transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/

Mega Menu Preview
Go into Wagtail admin and go to the Mega Menu settings and hit "Preview"

Prepaid agreements
http://localhost:8000/data-research/prepaid-accounts/search-agreements/

iRegs
http://localhost:8000/rules-policy/regulations/1002/

Before You Claim
http://localhost:8000/consumer-tools/retirement/before-you-claim/

TDP activities search
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/

Story page
http://localhost:8000/about-us/racial-equity/
http://localhost:8000/about-us/small-business-lending/
http://localhost:8000/es/quienes-somos/prestamos-pequenos-negocios/

Recent updates
http://localhost:8000/activity-log

Spanish homepage
http://localhost:8000/es/

Homepage
http://localhost:8000/

400 and 500 error pages

> 1. Temporarily change [this line](https://github.com/cfpb/consumerfinance.gov/blob/ef8522675c59e74f4d16ed86b0bc07b80bbcabec/cfgov/cfgov/settings/local.py#L18) to `False`.
> 2. Run `ALLOW_ADMIN_URL=1 cfgov/manage.py runserver --insecure`
> 3. To test the 404 template: visit http://localhost:8000/something-that-doesnt-exist/
> 4. To test the 500 template: visit http://localhost:8000/admin/failwhale/

Before you claim - Added block to content are to fix issue where there was no padding at the bottom of the content area.
http://localhost:8000/consumer-tools/retirement/before-you-claim/about/
